tools/doc-shredder/processors/shredMapProcessor.js

```diff
@@ -18,25 +18,37 @@ module.exports = function shredMapProcessor(log, createDocMessage) {
       var fragToJadeMap = {};
 
       docs.forEach(function(doc) {
-        var jadePath = path.join(options.jadeDir, doc.fileInfo.relativePath);
+        var relativePath = doc.fileInfo.relativePath;
+        var jadePath = path.join(options.jadeDir, relativePath);
+        var lang = relativePath.substr(0, relativePath.indexOf('\/'));
+        var appProjDirName = jadeBaseFileNameToExampleName(doc.fileInfo.baseName);
         var fragInfoSet = {};
         doc.fragItems.forEach(function(fragItem) {
           var mixinPath = fragItem.mixinPath;
           var fullExamplePath;
+          // Normalize mixinPath: strip out optional trailing '(...)'
+          var mixinPath = mixinPath.replace(/ \([^\)]*\)/,'');
           if ( mixinPath.indexOf('_api') >= 0) {
             var sourcePath = mixinPath.replace('_api/','');
             fullExamplePath = path.join(options.apiExamplesDir, sourcePath);
           } else {
             fullExamplePath = path.join(options.devguideExamplesDir, mixinPath);
           }
-          var region = fragItem.region ? "-" + fragItem.region : '';
-          var extn = path.extname(mixinPath);
-          var basename = path.basename(mixinPath, extn);
-          var fragDir = path.dirname(mixinPath);
-          var fragPath = path.join(fragDir, basename + region + extn) + '.md';
-          var fullFragPath =  path.join(options.fragmentsDir, fragPath);
-
-          var fragInfo = { fragPath: fullFragPath, examplePath: fullExamplePath, exists: fs.existsSync(fullFragPath) };
+          var fragInfo = makeFragInfo(options.fragmentsDir, fullExamplePath, fragItem, mixinPath);
```

Comment:
Can you explain what the consequences of this (and similar changes) are for the way we write examples in Jade?